### DISCUSSION TITLE
日本語への翻訳

### DISCUSSION THREAD

**Discussion by Shin-Marken** [aclap-dev/video-downloadhelper#1554] [Language & Translation]
一部ですが、ご協力致します。
[0d2809f9-f457-430d-b8c3-ba9e5e576684.txt](https://github.com/user-attachments/files/18164580/0d2809f9-f457-430d-b8c3-ba9e5e576684.txt)

**Comment by maboroshin** [comment]
**This is a new Japanese file, but it has problems and conflicts**

Only the following 2 strings are new and proper translations when compared to the August 2024 file below. Please adopt only the following 2 strings.
```
v9_settings_checkbox_thumbnail_in_notification = '''通知にサムネイルを表示する'''
v9_smartnaming_selector = '''CSS セレクターからのテキスト (オプション)'''
```

Because, this translation contains errors. The key names have been translated in this way.
```v9_badge_new = '''v9の新しいバッヂ'''```

Japanese files created in August 2024 exist: https://github.com/aclap-dev/video-downloadhelper/discussions/1138

**Comment by yomosan** [comment]
一応、翻訳して推敲してみました。
[C9qKjq5Qey48M9EzZTIA5nxmR.txt](https://github.com/user-attachments/files/19480983/C9qKjq5Qey48M9EzZTIA5nxmR.txt)

使いものになりますでしょうか。
